polus-graph-pyramid-builder-plugin/src/main.py

```diff
@@ -407,92 +862,120 @@ def metadata_to_graph_info(bins,outPath,outFile):
+                logger.info("What Y would be: " + str(subgrid_dims[0][y:y+2]))
+                if S==(info['scales'][0]['key'] - 5): #to use multiple processors to compute faster.
+                    sub_image = _get_higher_res_par(typegraph,
+                                                    S+1,
                                                    info,
+                                                   cnames,
                                                    outpath,
                                                    out_file,
+                                                   indexscale,
+                                                   indexdict,
+                                                   binstats,
+                                                   binsizes,
+                                                   axiszero,
+                                                   alphavals,
                                                    X=subgrid_dims[0][x:x+2],
                                                    Y=subgrid_dims[1][y:y+2])
                 else:
-                    sub_image = _get_higher_res(S+1,
+                    sub_image = _get_higher_res(typegraph,
+                                                S+1,
                                                info,
+                                               cnames,
                                                outpath,
                                                out_file,
+                                               indexscale,
+                                               indexdict,
+                                               binstats,
+                                               binsizes,
+                                               axiszero,
+                                               alphavals,
                                                X=subgrid_dims[0][x:x+2],
                                                Y=subgrid_dims[1][y:y+2])
-                # sub_image = _get_higher_res(S+1,
-                #                                 info,
-                #                                 outpath,
-                #                                 X=subgrid_dims[0][x:x+2],
-                #                                 Y=subgrid_dims[1][y:y+2])
-                
                 image[y_ind[0]:y_ind[1],x_ind[0]:x_ind[1],:] = _avg2(sub_image)
                 del sub_image
 
     # Write the chunk
     outpath = Path(outpath).joinpath('{}_files'.format(out_file),str(S))
     outpath.mkdir(exist_ok=True)
     imageio.imwrite(outpath.joinpath('{}_{}.png'.format(int(X[0]/CHUNK_SIZE),int(Y[0]/CHUNK_SIZE))),image,format='PNG-FI',compression=1)
-    logger.info('Finished building tile (scale,X,Y): ({},{},{})'.format(S,int(X[0]/CHUNK_SIZE),int(Y[0]/CHUNK_SIZE)))
+    logger.warning('Finished building tile (scale,X,Y): ({},{},{})'.format(S,int(X[0]/CHUNK_SIZE),int(Y[0]/CHUNK_SIZE)))
```

Comment:
This shouldn't be a warning. I suspect this was done for debugging purposes?

Reply:
Yes, but I just changed it to logger.debug now